aws/resource_aws_rds_cluster.go

```diff
@@ -650,6 +697,109 @@ func resourceAwsRDSClusterCreate(d *schema.ResourceData, meta interface{}) error
 			return err
 		}
 
+	} else if v, ok := d.GetOk("restore_to_point_in_time"); ok {
+		pointInTime := v.([]interface{})[0].(map[string]interface{})
+		createOpts := &rds.RestoreDBClusterToPointInTimeInput{
+			DBClusterIdentifier:       aws.String(identifier),
+			DeletionProtection:        aws.Bool(d.Get("deletion_protection").(bool)),
+			SourceDBClusterIdentifier: aws.String(pointInTime["source_cluster_identifier"].(string)),
+			Tags:                      tags,
+		}
+
+		if v, ok := pointInTime["restore_to_time"].(string); ok {
```

Comment:
we'll still need to check that the string here is not empty e.g.
```
if v, ok := pointInTime["restore_to_time"].(string); ok && v != ""{
```
since by default this `pointInTime` map stores the value here as the empty string even when this value is not explicitly configured in Terraform.

This line change here should clear up the current test failure that's returning:
```
  TestAccAWSRDSCluster_PointInTimeRestore: resource_aws_rds_cluster_test.go:374: Step 1/1 error: Error running apply:
        Error: InvalidParameterCombination: You cannot specify both UseLatestRestorableTime and RestoreToTime at same time.
        	status code: 400,
```